src/org/parosproxy/paros/view/MainMenuBar.java

```diff
@@ -210,6 +213,25 @@ public void run() {
 		}
 		return menuView;
 	}
+	/**
+
+	 * This method initializes menuImport
+
+	 *
+
+	 * @return javax.swing.JMenu
+
+	 */

```

Comment:
Could you undo the double spacing in this javadoc?

Reply:
No I mean the line spacing in the java doc should be single not double. (5 lines not 9).

Reply:
Well other menu javadocs are also in the same format. Should I undo it anyway?

Reply:
Oh weird, I have no idea why others are that way. Must be an old code thing. Yes, may as well do it clean. Thanks!

Reply:
So what should I do, remove the line spacing in javadoc for every function, or leave it as it is?

Reply:
I still don’t think we’re talking about the same thing...

![654ADAA5-2F4F-4284-A91B-AC1383698573](https://user-images.githubusercontent.com/7570458/54477218-26275580-47dc-11e9-8ea0-0330c6f9ef4d.jpeg)

Reply:
@kingthorin the question was about the existing code.

@Yashvendra leave it as is for existing functions (that will be fixed when we start to enforce the formatting), the one being added should be corrected though.